tools/nl/titles/demo_svs_1.3k_names.csv

```diff
@@ -2177,112 +2177,6 @@ Employed Population Below International Poverty Line With Gender = Male,dc/topic
 "Employed Population Below International Poverty Line With Age = 25 Years Old And Over, Gender",dc/topic/sdgSIPOVEMP1_age-SDGAgeEnumYGE25_gender,,Employed Population Aged 25 or Over Below International Poverty Line by Gender,,Employed Population Aged 25 or Over Below International Poverty Line by Gender
```

Comment:
What about dc/topic stuff, should we drop?

Reply:
dropped topics too!